nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [ ] Let's kindly raise the error this way:
`raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"`
[Pattern description](https://refactoring.guru/design-patterns/decorator/ruby/example#example-0)